src/pages/using-spark/guides/content-style-guidelines.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Content Style Guidelines
+---
+
+# Content Style Guidelines
+
+UX Writers, designers and developers can use the **Content Style Guidelines** as a guide to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+### Table Of Contents
+
+- [Capitalization](#capitalization)
+  - [Sentence Case](#sentence-case)
+  - [Start Case](#start-case)
+- [Quotation Marks](#quotation-marks)
+- [Accessibility Guidelines](#accessibility-guidelines)
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+### Start Case
+
+Use this case style to capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.
+
+#### Usage
+
+Use for:
+
+- Headers that are descriptive or functional
+- All subheads regardless of whether the primary header is title or sentence case
+- All buttons labels (Primary, Secondary and Tertiary Buttons)
+- Menu items
+- Chart and table headers
+- Referring to the name of a page
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
```

Comment:
The `docs-b-h4` class doesn't exist in our styles, so it can be removed here. There's some extra spaces in here, as well.

Reply:
I noticed in the raw code that @RVMendoza shared, the class **'docs-b-h1'** is added to the **h1 header**, and is also added to each header's tag, like in the h4. If it's not part of our styles, why does it get added?

Reply:
Looks like we do have classes for `docs-b-h2` && `docs-b-h3`, but not for `h1` or `h4`. Not entirely sure why we have those and not the others.